### DISCUSSION TITLE
Where did all the Typings go?

### DISCUSSION THREAD

**Discussion by CursedMun** [prisma/prisma#17944] [Q&A] [answered]
Some of the typings that prisma `4.9.0` had just disappeared in `4.10.1` without any mention in the release description and didn't find anything in `index.d.ts` to suit them.

Some of the typings I know that I miss
```
Prisma.TypeMap
Prisma.Exact
Prisma.Result
Prisma.Args
```

My question is: Is there any change to them or they are gone forever?

**Comment by CursedMun** [comment, marked as answer]
Nevermind, all good just forgot to add this previewFeatures
```prisma
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["fieldReference", "clientExtensions"] <- this one
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
